### ISSUE TITLE
Add features from the envy project

### ISSUE THREAD

**Issue by sametmax** [berdario/pew#106]
Envy (https://github.com/shaunvxc/envy) is an interesting project letting you edit a module in your env, then revert it back to it's pristine state, without using versioning tools. It's useful for debugging installed libs, and it's a perfect fit for a pew subcommand. After all, who wants to install a 100 virtualenv managing tools ?

**Comment by berdario** [comment]
Yeah, it's nice... but I don't think I'll implement this any time soon.
Is integrating it as a dependency of pew what you had in mind (similarly to what I did with Pythonz)?

**Comment by sametmax** [comment]
Nice idea.
